.github/workflows/tests.yml

```diff
@@ -143,6 +143,20 @@ jobs:
           count=$(ls *.json | wc -l | xargs echo -n)
           echo "$count bottles"
           echo "::set-output name=count::$count"
+          failures=$(ls failed/*.json | wc -l | xargs echo -n)
+          echo "$failures failed bottles"
+          echo "::set-output name=failures::$failures"
+
+      - name: Upload failed bottles
+        if: always() && steps.bottles.outputs.failures > 0
+        uses: actions/upload-artifact@main
+        with:
+          name: bottles (failed)
+          path: bottles/failed
+
+      - name: Delete failed bottles
+        if: always()
+        run: rm -rvf bottles/failed
```

Comment:
```suggestion
```

`Post Cleanup` will do this.

Reply:
This is done so that the `bottles` artifact doesn't pick this up

Reply:
Or should we include the `failed` subdirectory in the `bottles` artifact? (Instead of having a separate `bottles (failed)` artifact?)

Reply:
Doh! Fine with me as-is. Maybe add a comment to explain what I missed?